packages/xo-server/src/api/pool.mjs

```diff
@@ -334,6 +334,22 @@ getLicenseState.resolve = {
 
 // -------------------------------------------------------------------
 
+export async function getGuestSecureBootReadiness({ pool }) {
```

Comment:
This method should probably be not logged in the audit log: https://github.com/vatesfr/xen-orchestra/blob/0e4a300058a140341303e7c4c2a5fe16052a1df8/packages/xo-server-audit/src/index.js#L13